examples/laravel/composer.json

```diff
@@ -0,0 +1,64 @@
+{
+    "name": "laravel/laravel",
+    "description": "The Laravel Framework.",
+    "keywords": ["framework", "laravel"],
+    "license": "MIT",
+    "type": "project",
+    "require": {
+        "php": ">=7.0.0",
+        "fideloper/proxy": "~3.3",
+        "laravel/framework": "5.5.*",
+        "laravel/tinker": "~1.0",
+        "opencensus/opencensus": "dev-example-apps",
```

Comment:
It might be tricky, but I think we eventually want to change it from "dev-example-apps"

Reply:
Yeah, I needed to set it to something. I'll switch them to dev-master for now.